daemon/cmd/ciliumendpoints.go

```diff
@@ -87,16 +93,16 @@ func (d *Daemon) deleteCiliumEndpoint(
 			if k8serrors.IsNotFound(err) {
 				log.WithError(err).WithFields(logrus.Fields{logfields.CEPName: cepName, logfields.K8sNamespace: cepNamespace}).
 					Info("CEP no longer exists, skipping staleness check")
-			} else {
-				log.WithError(err).WithFields(logrus.Fields{logfields.CEPName: cepName, logfields.K8sNamespace: cepNamespace}).
-					Error("Failed to get possibly stale ciliumendpoints from apiserver, skipping.")
+				return nil
```

Comment:
We could so a bit of refactoring here and pull up the 

```
log.WithFields(logrus.Fields{
		logfields.CEPName:      cepName,
		logfields.K8sNamespace: cepNamespace,
```

a bit earlier and refactor the log statements which aren't DRY to use the logger fields from above.

Reply:
Thanks Chris! I saw that but figured it might be a bit out of scope for this pr??
I'll add the refactor in a separate commit.